.github/workflows/dockerpush.yml

```diff
@@ -11,21 +11,21 @@ env:
 
 
 jobs:
-  # Run tests.
-  # See also https://docs.docker.com/docker-hub/builds/automated-testing/
-  push-to-pypi-store:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python
-        uses: actions/setup-python@v1
-        with:
-          python-version: '3.x'
-      - name: Run 'make release'
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
-          KADALU_VERSION=${{ env.kadalu_version }} TWINE_PASSWORD=${{ secrets.TWINE_PASSWORD }} make release
+  # # Run tests.
```

Comment:
These are still needed I guess to upload kadalu-quotad package.

Reply:
make release failed with no docker login, is there a separate command to upload only kadalu-quotad package?

Reply:
For now, let me merge this and test out the script, once we have a separate command for quotad upload, I will include that here.

Reply:
Fixed! :+1: